presto-parquet/src/main/java/com/facebook/presto/parquet/reader/MetadataReader.java

```diff
@@ -283,6 +289,8 @@ private static OriginalType getOriginalType(ConvertedType type)
                 return OriginalType.JSON;
             case BSON:
                 return OriginalType.BSON;
+            case TIMESTAMP_MICROS:
```

Comment:
Can you put this change in a separate commit? This commit should only upgrade Parquet, and in a subsequent commit we can handle the timestamp type.

Reply:
Removed.